Testnet/DAOContract/README.MD

```diff
@@ -0,0 +1,15 @@
+﻿# DAO Smart Contract
+
+**Compiler Version**
+```
+v2.0.0
+```
+**Contract Hash**
+```
+6ffa1efa86dba93e86222df73b07d4fcae32b7e7c79282c96264af1264a939d4
```

Comment:
Please update the bytecode across Mainnet, Testnet README's and the PR description.